src/OgAccess.php

```diff
@@ -191,7 +191,8 @@ public static function userAccessEntity($operation, EntityInterface $entity, Acc
       foreach ($entity_groups as $groups) {
         foreach ($groups as $group) {
           $user_access = static::userAccess($group, $operation, $user);
-          if ($user_access->isAllowed()) {
+          if (!$user_access->isForbidden()) {
+            //this covers allowed and neutral
             return $user_access->addCacheTags($cache_tags);
```

Comment:
The coding standards are very specific for comments, there should be a space after the `//`, the first word should be capitalized, and the sentence should end in a period. Painful, I know ;)